### PR TITLE
VxDesign: Seal configuration

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

This PR bundles a few changes that were difficult to separate out:
- Require `Election.seal` (inline SVG text contents - the way we used to do seals a while back)
- Remove `Election.sealUrl` (a path pointed to a seal image asset that's committed to this repo, which we no longer want to support)
- Add a custom seal field to the ballot definition CDF
- Add a form to VxDesign to upload a seal SVG image and export it in the election definition

_Review by commits_

## Demo Video or Screenshot

https://github.com/votingworks/vxsuite/assets/530106/2a4cca85-ef16-4dd8-8241-c8ed5dd524d3

## Testing Plan
- Updated automated tests
- Confirmed seals render correctly in ballot fixtures
- Confirmed seals show correctly in apps 

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/hmpb/render-backend/src/render_ballot.tsx [hunk 1]
```diff
@@ -16,20 +16,38 @@ import {
...
+  parsed.documentElement.removeAttribute('height');
+  return new XMLSerializer().serializeToString(parsed);
+}
+
 // SVG-to-PDFKit doesn't support embedded SVGs, so we hack around it by
 // inlining the SVG contents.
-function InlineSvgImage(props: SvgImageProps): JSX.Element {
-  const imageContents = readFileSync(join(ASSETS_DIR, props.href)).toString(
-    'utf8'
+function InlineSvgImage({
```
Comment (arsalansufi) [inline libs/hmpb/render-backend/src/render_ballot.tsx:36]: Mind refreshing my memory on what the relationship is between `SvgImage` in `libs/hmpb/layout/src/document_svg.tsx` and `InlineSvgImage` here in `libs/hmpb/render-backend/src/render_ballot.tsx`, if any?

File: apps/design/frontend/src/election_info_screen.tsx [hunk 2]
```diff
@@ -106,9 +110,53 @@ function ElectionInfoForm({
...
+                }
+              };
+              reader.readAsText(file);
+            }}
+            disabled={!isEditing}
+          >
+            Upload Seal Image
+          </FileInputButton>
+        )}
+      </FormField>
```
Comment (arsalansufi) [inline apps/design/frontend/src/election_info_screen.tsx:148]: Would be curious to see what the form + form interaction looks like. Not blocking and I'm happy to wait for a more holistic tour of the latest UI later on

**Review by arsalansufi — APPROVED** [review]

File: libs/hmpb/render-backend/src/render_ballot.tsx (hunk 1 above)
Comment (jonahkagan) [reply libs/hmpb/render-backend/src/render_ballot.tsx:36]: `SvgImage` renders for the browser, `InlineSvgImage` for PDF. It's the one place where the rendering needs to diverge due to peculiarities with the PDF tooling.

File: apps/design/frontend/src/election_info_screen.tsx (hunk 2 above)
Comment (jonahkagan) [reply apps/design/frontend/src/election_info_screen.tsx:148]: Oh duh! Got so into the habit of refactors that I forgot to add a video lol

Comment (jonahkagan) [reply apps/design/frontend/src/election_info_screen.tsx:148]: Just added to PR description

File: libs/hmpb/render-backend/src/render_ballot.tsx (hunk 1 above)
Comment (jonahkagan) [reply libs/hmpb/render-backend/src/render_ballot.tsx:36]: Though I suppose I could probably converge on the method used by `InlineSvgImage` now that all seals are just text content and remove support for `href`. I will experiment with that in a follow up PR

File: apps/design/frontend/src/election_info_screen.tsx (hunk 2 above)
Comment (arsalansufi) [reply apps/design/frontend/src/election_info_screen.tsx:148]: Haha makes sense - the UX looks good!

**Pull request merged**